int32   velocity_left
int32   velocity_right
int32   odometry_left
int32   odometry_right
